src/hooks/useHandleNFTRedeem.tsx

```diff
@@ -111,7 +111,7 @@ async function solana(
     if (!claimInfo) {
       await postVaaSolanaWithRetry(
         connection,
-        wallet.signTransaction,
+        wallet.signTransaction.bind(wallet),
```

Comment:
I'm curious, why do you need to do this?

Reply:
The `bind` is needed in order to prevent the `this` variable from being undefined when calling the wallet's `signTransaction` method. I could alternatively go with an anonymous function `(transaction) => wallet.signTransaction(transaction)` if it's clearer.

The previous wallet does not make use of `this`, so it wasn't needed (see implementation [here](https://github.com/solana-labs/wallet-adapter/blob/master/packages/core/react/src/WalletProviderBase.tsx#L207))

Reply:
oh, I see, yup, you are passing the function reference, I think, yes an anonymous arrow function would make it simple to understand, but, the current approach is okay too, lets wait for the other folks' opinion